README.md

```diff
@@ -1,63 +1,3280 @@
+```
+As you can see, the certificate is again not valid as the current device time
+does not fall within the validity window, and so the Supervisor won't pull the
+updated application. If we restart chrony, this will be rectified and the
+Supervisor will, after a short delay untill its next poll, update the
+application:
+```
+root@f34c2e9:~# systemctl start chronyd.service
+root@f34c2e9:~# journalctl -f -u resin-supervisor
+-- Logs begin at Thu 2017-03-23 12:00:04 UTC. --
+Mar 23 12:01:00 f34c2e9 resin-supervisor[1830]: [error]   Failed to get target state for device: Error: certificate is not yet valid
+Mar 23 12:01:30 f34c2e9 resin-supervisor[1830]: [error]   Failed to get target state for device: Error: certificate is not yet valid
+Mar 23 12:02:30 f34c2e9 resin-supervisor[1830]: [error]   Failed to get target state for device: Error: certificate is not yet valid
+Mar 23 12:03:30 f34c2e9 resin-supervisor[1830]: [event]   Event: Update notification {}
+Mar 23 12:03:30 f34c2e9 resin-supervisor[1830]: [api]     POST /v1/update 204 - 26.804 ms
+Mar 23 12:03:31 f34c2e9 resin-supervisor[1830]: [error]   Failed to get target state for device: Error: certificate is not yet valid
+Mar 23 12:04:23 f34c2e9 resin-supervisor[1830]: [debug]   Attempting container log timestamp flush...
+Mar 23 12:04:23 f34c2e9 resin-supervisor[1830]: [debug]   Container log timestamp flush complete
+Mar 23 12:04:26 f34c2e9 resin-supervisor[1830]: [api]     GET /v1/healthy 200 - 10.975 ms
+Mar 23 12:04:31 f34c2e9 resin-supervisor[1830]: [error]   Failed to get target state for device: Error: certificate is not yet valid
+Nov 22 11:59:30 f34c2e9 resin-supervisor[1830]: [info]    Waiting for connectivity...
+Nov 22 11:59:50 f34c2e9 resin-supervisor[1830]: [info]    Internet Connectivity: OK
+Nov 22 12:02:17 f34c2e9 resin-supervisor[1830]: [info]    Applying target state
+Nov 22 12:02:18 f34c2e9 resin-supervisor[1830]: [debug]   Replacing container for service frontend because of config changes:
+Nov 22 12:02:18 f34c2e9 resin-supervisor[1830]: [debug]     Non-array fields:  {"added":{},"deleted":{"command":{},"entrypoint":{},"environment":{},"labels":{}},"updated":{"image":"registry2.balena-cloud.com/v2/83732ad30648d5f3e68188ed154ffdc2@sha256:8ec0ba1f96a51d936a83d8aa0ac1db2827a0458c2b6f7e3a5f814d6ad26b6d1f","workingDir":""}}
+...
+```
+This shows the importance of a working service such as timesetting, and how this
+can affect the system as a whole. As a note, be aware that not *every* device
+relies completely on NTP. Some devices, such as an Intel NUC, also have battery
+backed services including an RTC, which means that even if NTP is not working
+the time may look at first glance as though it's correct. However, if NTP is
+not operational even an RTC will eventually suffer from clock skew (the slow
+movement away from the real time due to drift) which will eventually cause
+issues.
```

Comment:
Is it worth mentioning the HTTP method of updating the clock in the case that the NTP is block and one needs to get the device to a nominal state?

Reply:
One day @ZubairLK will build htpdate into the hostOS :)